### DISCUSSION TITLE
如何缩小pdf大小

### DISCUSSION THREAD

**Discussion by jiaqiwang969** [sjtug/SJTUThesis#852] [提问]
如何通过Latex输出压缩版的pdf？
当前为了查重要求，需要将pdf压缩到50M以下。但用pdf压缩，只能最小到80M。是否在Latex编译层面，压缩pdf

**Comment by LogCreative** [comment]
一般而言，PDF文件体积过大是由于插入的图片体积较大，可以检查一下插入的图片大小，减少过大图片的分辨率。
